projects/plugins/jetpack/sal/class.json-api-post-base.php

```diff
@@ -1017,11 +1017,9 @@ private function get_media_item_v1_1( $media_id ) {
 					}
 				}
 
-				$response['videopress_guid']            = $info->guid;
+				$response['videopress_guid']            = $info->guid ?? null;
```

Comment:
I'm wondering if this should be `is_object($info) ? $info->guid ?? null : null` ? If guid is still a bool it would seem like this would still trigger a warning as we're trying to access a property on a bool?

Reply:
Ah true, it doesn't actually generate a PHP warning using isset, only editor notices in test cases.